manifests/cluster-rbac/argocd-server-clusterrolebinding.yaml

```diff
@@ -13,4 +13,3 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: argocd-server
-  namespace: argocd
```

Comment:
Curious, does this work?

Reply:
Yep! We have test installation in `argocd-test` namespace.

These parts related to this PR:
```
# ... kustomization.yaml ...
---
namespace: argocd-test

resources:
  - github.com/argoproj/argo-cd/manifests/cluster-install?ref=v1.8.7
patchesStrategicMerge:
  - patch.fix-rolebindings.yaml

# ... patch.fix-rolebindings.yaml ....
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: argocd-server
subjects:
  - kind: ServiceAccount
    name: argocd-server
    namespace: argocd-test
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: argocd-application-controller
subjects:
  - kind: ServiceAccount
    name: argocd-application-controller
    namespace: argocd-test
```